src/frontend/packages/core/src/shared/components/page-header/page-header.component.ts

```diff
@@ -190,7 +192,13 @@ export class PageHeaderComponent implements OnDestroy, AfterViewInit {
       map(dashboardState => dashboardState.gravatarEnabled)
     );
 
-    this.canAPIKeys$ = this.cups.can(StratosCurrentUserPermissions.API_KEYS);
+    // Must be enabled and the user must have permission
+    this.canAPIKeys$ = combineLatest([
+      this.endpointsService.disablePersistenceFeatures$.pipe(startWith(true)),
```

Comment:
I think we should fold this check into the permissions model. In `StratosUserPermissionsChecker` we could add a `StratosPermissionTypes.REQUIRES_PERSISTENCE` to `getSimpleCheck` as per `StratosPermissionTypes.API_KEY`. This would then return the opposite of `endpointsService.disablePersistenceFeatures$`. Would then need to just add the new type to `StratosCurrentUserPermissions.API_KEYS` alongside `PermissionConfig(StratosPermissionTypes.API_KEY`

Reply:
Ignore, permission model reflects user orientated checks rather than functional